MaxText/train.py

```diff
@@ -71,6 +71,7 @@
 Transformer = models.Transformer
 EPS = 1e-8
 _DEFAULT_OCDBT_TARGET_DATA_FILE_SIZE = 2 * 1024**3
+_DEFAULT_TOLERANCE = 0.02
```

Comment:
can you move this to a field in base.yml named e.g. "sharded_tolerance" defaulting to 0.02